src/library_wasmfs.js

```diff
@@ -1,3 +1,65 @@
-var WasmfsLibrary = {}
+var WasmfsLibrary = {
+  $FS : {
+    preloadedFiles: [],
+    preloadedDirs: [],
+    // TODO: Clean up the following functions - currently copied from library_fs.js directly.
+    createPreloadedFile: function(parent, name, url, canRead, canWrite, onload, onerror, dontCreateFile, canOwn, preFinish) {
```

Comment:
```suggestion
  $wasmFS$preloadedFiles: [],
  $wasmFS$preloadedDirs: [],
  $FS__deps: ['$wasmFS$preloadedFiles', '$wasmFS$preloadedDirs'],'
  $FS : {
    // TODO: Clean up the following functions - currently copied from library_fs.js directly.
    createPreloadedFile: function(parent, name, url, canRead, canWrite, onload, onerror, dontCreateFile, canOwn, preFinish) {
```

Reply:
This would make the code more easily DCE-able as @juj suggested.

Will need to add the `wasmFS$` prefix in uses of them too.